apps/condo/domains/meter/schema/RegisterMetersReadingsService.test.js

```diff
@@ -928,4 +928,48 @@ describe('RegisterMetersReadingsService', () => {
             expect(meter.controlReadingsDate).toBe(output ? dayjs(output).toISOString() : output)
         })
     })
+
+    test('prevent to create readings duplicates', async () => {
+        const [o10n] = await createTestOrganization(adminClient)
```

Comment:
I’m not sure exactly, but it seems @SavelevMatthew  wrote that we should not use the abbreviation o10n since it is not a generally accepted abbreviation like i18n -> internationalization. And for other o10n developers it is not obvious that this is an organization